include/gatekeeper_net.h

```diff
@@ -125,8 +125,8 @@ struct gatekeeper_if {
 	uint16_t        num_rx_queues;
 	uint16_t        num_tx_queues;
 
-	/* The total burst size of any functional block for this interface. */
-	uint16_t        total_pkt_burst;
+	/* The maximum burst size of any functional block for this interface. */
+	uint16_t        max_pkt_burst;
```

Comment:
We need the total, not the maximum. Once each block is creating its own mbuf pool, they will know exactly what they need. A block may decide that it doesn't need a pool or it needs a smaller pool. For example, the LLS block needs a pool when there is no hardware support for its filters at one of the interfaces. On this occasion, the LLS block would still add its needed packets to field `total_pkt_burst`, so the GK or GT blocks have enough packets to receive and forward the packets to the LLS block.